packages/flutter/test/material/text_form_field_test.dart

```diff
@@ -1154,7 +1178,7 @@ void main() {
 
     await tester.pumpWidget(MaterialApp(
       home: Scaffold(
-        body: TextField(
+        body: TextFormField(
```

Comment:
Bonus: I just noticed that this adjacent test was testing TextField when it meant to test TextFormField.